tool/src/webapp/js/show-page.js

```diff
@@ -1794,6 +1794,10 @@ $(function() {
 			delbutton = $('#delete-student-item');
 			return delete_confirm(event, msg("simplepage.deletestudentsubmissionexist"));
 		    });
+		$('#delete-item, #delete-multimedia-item, #delete-youtube-item, #delete-movie-item, #delete-question-item').click(function(event) {
```

Comment:
because this .deleteButton class is also used by '#delete-comments-item' and '#delete-student-item' divs which have separate delete confirmation message .